bentoml/_internal/runner/strategy.py

```diff
@@ -37,7 +35,7 @@ def setup_worker(
     "OMP_NUM_THREADS",
     "TF_NUM_INTEROP_THREADS",
     "TF_NUM_INTRAOP_THREADS",
-]
+]  # TODO(jiang): make it configurable?
```

Comment:
If users want to configure this, wouldn't they just change the environment value?